core/src/main/java/io/grpc/internal/ManagedChannelImpl.java

```diff
@@ -1806,12 +1813,17 @@ public Status onResult2(final ResolutionResult resolutionResult) {
         }
         Attributes attributes = attrBuilder.build();
 
-        return helper.lb.tryAcceptResolvedAddresses(
-            ResolvedAddresses.newBuilder()
-                .setAddresses(servers)
-                .setAttributes(attributes)
-                .setLoadBalancingPolicyConfig(effectiveServiceConfig.getLoadBalancingConfig())
-                .build());
+        ResolvedAddresses.Builder resolvedAddresses = ResolvedAddresses.newBuilder();
+        if (serversOrError != null && serversOrError.hasValue()) {
```

Comment:
It looks like this is currently guaranteed to be `true`, because of the early return.

Reply:
Done.